.github/scripts/verifyServerStart.sh

```
@@ -0,0 +1,39 @@
+  echo "Server did not become healthy within $timeout seconds."
+  kill $NODE_PID
+  exit 1
+fi
+
+# 3. Kill the node server with SIGINT
+kill -SIGINT $NODE_PID
+
+# 4. Wait until /health no longer responds, up to 10 seconds
+echo "Waiting for server to stop..."
+stoptimeout=10
+for i in $(seq 1 $stoptimeout); do
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)